tools/packaging/kata-deploy/runtimeclasses/kata-remote.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1
+metadata:
+    name: kata-remote
+handler: kata-remote
+overhead:
+    podFixed:
+        memory: "2048Mi"
+        cpu: "1.0"
```

Comment:
So I will fully admit that I don't think a look of science has gone into measuring the actual kata-remote overhead yet (so I'd maybe ad a comment with this caveat), but the values that were picking in the webhook hack are 120Mi of RAM and 250mCPUs [from here](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/webhook/hack/rc.yaml). These might be out of date, so I've raised https://github.com/confidential-containers/cloud-api-adaptor/issues/1309 to review them and reflect that here once the next set of refactors are done, but they are the only (and therefore best) suggestion we currently have.

Reply:
Ack, adjusted, thanks for taking a look at this!